src/config.c

```diff
@@ -88,6 +88,8 @@ enum blur_method parse_blur_method(const char *src) {
 		return BLUR_METHOD_BOX;
 	} else if (strcmp(src, "gaussian") == 0) {
 		return BLUR_METHOD_GAUSSIAN;
+	} else if (strcmp(src, "dual_kawase") == 0) {
+		return BLUR_METHOD_DUAL_KAWASE;
```

Comment:
As my first implementation for compton used the slightly incorrect name "kawase" for this parameter it might be a good idea to accept it as well and output a deprecation message.

Reply:
Agreed